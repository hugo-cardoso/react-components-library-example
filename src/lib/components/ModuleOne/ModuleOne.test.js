import React from "react";
import ModuleOneComponent from "./ModuleOne.component";
import renderer from "react-test-renderer";

describe("TextInput", () => {
  it("renders properly", () => {
    const tree = renderer
      .create(<ModuleOneComponent title="Hugo"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
