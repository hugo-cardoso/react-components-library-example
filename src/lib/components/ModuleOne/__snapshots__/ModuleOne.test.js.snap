// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders properly 1`] = `
<h1>
  Hugo
</h1>
`;
